lib/service/endpoint.mli

```diff
@@ -168,7 +168,7 @@ val outer
 val href
   :  ?anchor:string
   -> ?parameters:(string * string) list
-  -> ('scope_, [< Method.for_link ], 'continuation, string) wrapped
```

Comment:
And here, and for the following, what is the difference? Why explicitely discards the type variable name?

Reply:
Just because `'scope_` sound weird (IMO).